lib/pup/dependencies/string.rb

```diff
@@ -0,0 +1,13 @@
+class String
+  def to_camelcase
+    self.split('_').map(&:capitalize).join
```

Comment:
Redundant `self` detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.